Tests/Kernel/fuzz-syscalls.cpp

```diff
@@ -86,7 +86,6 @@ static void randomize_from(size_t* buffer, size_t len, const Vector<size_t>& val
 }
 
 // The largest SC_*_params struct is SC_mmap_params with 36 bytes.
```

Comment:
is this still correct?

Reply:
Nope, it sure isn't. Changing that to use `sizeof()`.